### ISSUE TITLE
`trivy config` Template Output Produces Blank Results

### ISSUE THREAD

**Issue by AErmie** [aquasecurity/trivy#1317]
## Description

When using `trivy config` to scan a Dockerfile, even though the console shows output results, using the `--format template --template "@contrib/html.tpl" --output /output/TrivyIAC-Results.html` output does not capture any results. 

**Console output**
![Trivy-Dockerfile-Console-Output](https://user-images.githubusercontent.com/10376796/138163920-7f6a32e9-5d8e-4dc0-9965-97f03161a955.png)

**File output (HTML example)**
![Trivy-Dockerfile-HTML-Output](https://user-images.githubusercontent.com/10376796/138163929-46e2c1fc-abf2-47d6-9da1-590977046606.png)

## What did you expect to happen?

I expected the detected vulnerabilities would be included in the outputs.

## What happened instead?

All outputs (XML, HTML, SARIF), are empty.

## Output of run with `-debug`:

``` CMD
2021-10-20T19:49:32.152Z	DEBUG	Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
2021-10-20T19:49:32.155Z	DEBUG	cache dir:  /root/.cache/trivy
2021-10-20T19:49:32.155Z	DEBUG	Vulnerability type:  []
2021-10-20T19:49:32.281Z	DEBUG	OS is not detected and vulnerabilities in OS packages are not detected.
2021-10-20T19:49:32.281Z	INFO	Detected config files: 1
2021-10-20T19:49:32.281Z	DEBUG	Scanned config file: Dockerfile

Dockerfile (dockerfile)
=======================
Tests: 23 (SUCCESSES: 22, FAILURES: 1, EXCEPTIONS: 0)
Failures: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 1, CRITICAL: 0)
```

## Output of `trivy -v`:

```
Version: 0.20.1
Vulnerability DB:
  Type: Full
  Version: 1
  UpdatedAt: 2021-10-20 18:05:04.975582223 +0000 UTC
  NextUpdate: 2021-10-21 00:05:04.975581823 +0000 UTC
  DownloadedAt: 2021-10-20 18:10:43.785377252 +0000 UTC
```

## Additional details (base image name, container registry info...):

Base Docker image is Node v14

**Comment by afdesk** [comment]
Hi @AErmie thanks for your interest in `trivy`.
I couldn't reproduce your issue.
my actions:
```sh
$ trivy -d config --format template --template "@contrib/html.tpl" --output TrivyIAC-Results.html .

2021-11-24T13:45:31.347+0600	DEBUG	Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
2021-11-24T13:45:31.349+0600	DEBUG	cache dir:  /home/user/.cache/trivy
2021-11-24T13:45:31.349+0600	DEBUG	Vulnerability type:  []
2021-11-24T13:45:31.486+0600	DEBUG	OS is not detected and vulnerabilities in OS packages are not detected.
2021-11-24T13:45:31.487+0600	INFO	Detected config files: 1
2021-11-24T13:45:31.488+0600	DEBUG	Scanned config file: Dockerfile
```
and `TrivyIAC-Results.html` contains the result. 

**note**: the current dir should have `contrib` folder with templates.

**Comment by afdesk** [comment]
@AErmie can i help you with something else?

**Comment by afdesk** [comment]
oh @AErmie so sorry, it was my fault, I reproduced this issue. 
and we'll fix it.
thanks for the report!

**Comment by afdesk** [comment]
@AErmie 
we added misconfiguration fields to html template: [https://github.com/aquasecurity/trivy/blob/main/contrib/html.tpl](https://github.com/aquasecurity/trivy/blob/main/contrib/html.tpl)

**Comment by jpinkham** [comment]
This seems resolved as of v0.22.0
<img width="1440" alt="Trivy_HTML_output_asof_v0 22 0" src="https://user-images.githubusercontent.com/2560080/148664786-d6fd9f42-0103-4f66-87e0-711cf8f95a17.png">

**Comment by AErmie** [comment]
Thanks @afdesk, I see the HTML report is showing the values now. Appreciate the effort.
